invenio_app_ils/templates/invenio_app_ils/backoffice.html

```diff
@@ -1,30 +1,25 @@
-# -*- coding: utf-8 -*-
-#
-# Copyright (C) 2018 CERN.
-#
-# invenio-app-ils is free software; you can redistribute it and/or modify it
-# under the terms of the MIT License; see LICENSE file for more details.
+{# -*- coding: utf-8 -*-
```

Comment:
What's happening here? :)

Reply:
without these brackets in the start it was rendered in the html ;) Now it's recognized as a comment.